i18n/Aliases/Aliases.php

```diff
@@ -2,9 +2,9 @@
 $specialPageAliases = [];
 
 $specialPageAliases['en'] = [
-	'ApproverCategories' => [ 'Approver Categories' ],
+	'ApproverCategories' => [ 'ApproverCategories' ],
```

Comment:
We can change this later: ManageApprovers or similar is better. Makes it clear it is a management UI, not just a list. And it allows us to add additional info to approvers at some future point, without the name becoming weird.